### PR TITLE
Filebeat as DaemonSet for 'k8s as cloud service'

### PR DESCRIPTION
**Pull request by rpudlowski93**
Related to task #1670 

- Filebeat as DaemonSet added to filebeat role
- Helm and archived chart used for installation
- Small necessary changes added to repository role ( requirements )
- Added environment with kubeconfig to filebeat playbook in order to use helm locally

### CODE REVIEW

File: CHANGELOG-0.8.md [hunk 1]
```diff
@@ -15,7 +15,8 @@
 - [#1225](https://github.com/epiphany-platform/epiphany/issues/1225) - Add OS_PATCHING.md with information about patching RHEL OS
 - [#1656](https://github.com/epiphany-platform/epiphany/issues/1656) - Run Helm tasks from Epiphany container
 - [#1640](https://github.com/epiphany-platform/epiphany/issues/1640) - Added separate machine for repository and changed helm to use localhost address
-- [#1640](https://github.com/epiphany-platform/epiphany/issues/1673) - Added Node Exporter as DaemonSet for Kubernetes as Cloud Service
+- [#1640](https://github.com/epiphany-platform/epiphany/issues/1673) - Added Node Exporter as DaemonSet for "Kubernetes as Cloud Service"
+- [#1640](https://github.com/epiphany-platform/epiphany/issues/1670) - Added Filebeat as DaemonSet for "Kubernetes as Cloud Service"
```
Comment (atsikham) [inline CHANGELOG-0.8.md:19]: Wrong link text

File: CHANGELOG-0.8.md [hunk 2]
```diff
@@ -15,7 +15,8 @@
 - [#1225](https://github.com/epiphany-platform/epiphany/issues/1225) - Add OS_PATCHING.md with information about patching RHEL OS
 - [#1656](https://github.com/epiphany-platform/epiphany/issues/1656) - Run Helm tasks from Epiphany container
 - [#1640](https://github.com/epiphany-platform/epiphany/issues/1640) - Added separate machine for repository and changed helm to use localhost address
-- [#1640](https://github.com/epiphany-platform/epiphany/issues/1673) - Added Node Exporter as DaemonSet for Kubernetes as Cloud Service
+- [#1640](https://github.com/epiphany-platform/epiphany/issues/1673) - Added Node Exporter as DaemonSet for "Kubernetes as Cloud Service"
```
Comment (atsikham) [inline CHANGELOG-0.8.md:18]:
```suggestion
- [#1673](https://github.com/epiphany-platform/epiphany/issues/1673) - Added Node Exporter as DaemonSet for "Kubernetes as Cloud Service"
```

File: CHANGELOG-0.8.md (hunk 1 above)
Comment (atsikham) [inline CHANGELOG-0.8.md:19]:
```suggestion
- [#1670](https://github.com/epiphany-platform/epiphany/issues/1670) - Added Filebeat as DaemonSet for "Kubernetes as Cloud Service"
```

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/main.yml [hunk 3]
```diff
@@ -9,7 +14,7 @@
   when:
     - groups['kibana'][0] is defined
 
-- name: Include auditd configuration tasks
+- name: Include auditd configuration tasks
```
Comment (atsikham) [inline core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/main.yml:17]:
```suggestion
- name: Include auditd configuration tasks
```

**Review by atsikham — CHANGES_REQUESTED** [review]

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-daemonset.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        tasks_from: download_file
+      vars:
+        file_name: "{{ exporter_chart_file_name }}"
+        repository_url: http://localhost/epirepo
+
+    - name: Install Filebeat as DaemonSet using custom config file
+      block:
+        - name: Set Helm chart values from custom configuration
+          set_fact:
+            _helm_chart_values: "{{ lookup('template', '../templates/custom-chart-values.j2') }}"
```
Comment (rafzei) [inline core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-daemonset.yml:29]: Do we need this lookup there?

File: CHANGELOG-0.8.md (hunk 1 above)
Comment (rpudlowski93) [reply CHANGELOG-0.8.md]: Fixed, thanks

File: CHANGELOG-0.8.md (hunk 2 above)
Comment (rpudlowski93) [reply CHANGELOG-0.8.md]: Fixed, thanks

File: CHANGELOG-0.8.md (hunk 1 above)
Comment (rpudlowski93) [reply CHANGELOG-0.8.md]: Fixed, thanks

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/main.yml (hunk 3 above)
Comment (rpudlowski93) [reply core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/main.yml]: Fixed, thanks

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-daemonset.yml (hunk 4 above)
Comment (rpudlowski93) [reply core/src/epicli/data/common/ansible/playbooks/roles/filebeat/tasks/install-filebeat-as-daemonset.yml]: True, it can be done more easily. Done :)

**Review by rafzei — APPROVED** [review]
LGTM

**Review by atsikham — APPROVED** [review]

**Pull request merged**